docs/scos/dev/the-docker-sdk/202204.0/configure-services.md

```diff
@@ -564,6 +567,45 @@ class MonitoringDependencyProvider extends SprykerMonitoringDependencyProvider
 }
 ```
 
+Next create the class `src/Pyz/Service/NewRelic/Plugin/NewRelicMonitoringExtensionPlugin.php`
+
+```php
+<?php
+​
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+​
+namespace Pyz\Service\NewRelic\Plugin;
+​
+use SprykerEco\Service\NewRelic\Plugin\NewRelicMonitoringExtensionPlugin as SprykerNewRelicMonitoringExtensionPlugin;
+​
+class NewRelicMonitoringExtensionPlugin extends SprykerNewRelicMonitoringExtensionPlugin
+{
+    /**
+     * @param string|null $application
+     * @param string|null $store
+     * @param string|null $environment
+     *
+     * @return void
+     */
+    public function setApplicationName(?string $application = null, ?string $store = null, ?string $environment = null): void
+    {
+        if (!$this->isActive) {
+            return;
+        }
+​
+        // Custom application environment name, or use $environment as fallback
+        $environment = getenv('NEWRELIC_CUSTOM_APP_ENVIRONMENT') ?? $environment;
```

Comment:
The `getenv` function cannot return `null`, therefore if the env variable `NEWRELIC_CUSTOM_APP_ENVIRONMENT` is unset, the result will be `false` and `$environment` won't be taken into account.
```suggestion
        $environment = getenv('NEWRELIC_CUSTOM_APP_ENVIRONMENT') ?: $environment;
```

Reply:
Thank you @peresypkin for the suggestion! Change made